src/account/operations/transaction/high_level/minting/mod.rs

```diff
@@ -1,5 +1,6 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+pub(crate) mod mint_more_native_token;
```

Comment:
Thinking out loud, don't mind me: I kinda feel like this should be better named `increaseNativeTokenSupply` but then melt should also be `decreaseNativeTokenSupply` which I agree would be different to the TIP (do we even care?). I just find it a tiny bit weird of a name atm.

Reply:
Also not really happy with the naming